src/sources/internal_logs.rs

```diff
@@ -99,17 +100,21 @@ mod tests {
 
     #[tokio::test]
     async fn receives_logs() {
+        let test_id: u8 = rand::random();
         let start = chrono::Utc::now();
```

Comment:
Could you just use this start time to provide a unique-enough ID? It contains at least a microsecond portion.

Reply:
I liked this idea initially, but it doesn't actually seem to simplify much. We'd need to stringify the timestamp to pass it as a field, and doing so in a way that's consistent and precise enough to be unique is a bit more than I'd want to inline all three places. So the end result is the same number of lines, a bit more complicated, and brings back the original concern of timing collisions.

Reply:
Fair enough.